### DISCUSSION TITLE
SSR dev and build with errors!

### DISCUSSION THREAD

**Discussion by tarikrital** [quasarframework/quasar#16306] [CLI - SSR mode] [answered]
Welcome, I'm using Quasar V2 SSR mode but I get `load module script` error in both dev mode and build using a local server!

`yarn quasar build -m ssr`

```
Error: [vite]: Rollup failed to resolve import "/assets/index.c9b40fa0.js" from "index.html".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`

```

**Comment by tarikrital** [comment, marked as answer]
I found the issue depends on the `index.html` which includes link to a JS file.

**Reply by rstoenescu** [reply]
No reason to manually specify such a tag in index.html, unless you have a reaaaally specific requirement.

**Reply by tarikrital** [reply]
Yes, I have GA added.

Thank you, for all the framework jobs and the Quasar team too.

**Reply by rstoenescu** [reply]
As long as you refer to something from a CDN it's OK. Otherwise, referring to local files is not OK.
